examples/sandbox/README.md

```diff
@@ -0,0 +1,29 @@
+# Sandbox
+
+This example is a sandbox for local development. It sets up a basic page
+using Search UI.
+
+It is wired up to use the local source of the search-ui libraries as dependencies via lerna.
+This means as you update library code you can see it updated live in this project.
+
+It is set up to use a pre-configured Elastic App Search engine.
+
+## Using
+
+To run:
+
+```
+# Bootstrapping with lerna will install all dependencies for this entire
+# repository.
+(cd ../.. && npm run bootstrap)
+
+# Start watching search-ui js for changes in the various library repositories
+(cd ../.. && npm run watch-js)
+
+# Run the web application
+npm start
```

Comment:
I couldn't find any reference to `npm start` when looking through all of the `package.json` files until I ran a global search for "start" and realized that there is a new `examples` directory with the sandbox inside.

This is just related to the need for extra description around the structure of the mono-repo and how it all plays together.